src/content/docs/apm/agents/net-agent/installation/install-net-agent-windows.mdx

```diff
@@ -183,9 +183,13 @@ Available options are:
 
 ## Enable the agent for apps not in IIS [#enabling-the-agent]
 
-If the application you want to monitor is not hosted in IIS, you will follow the [standard install procedure](#msi) but in addition you must explicitly enable the agent for that application. Enabling the agent differs for .NET Framework and .NET Core applications.
+If the application you want to monitor is not hosted in IIS, you'll follow the [standard install procedure](#msi), with some additional caveats and action items: 
 
-When following the [standard install procedure](#msi), we recommend that you enable **Instrument all**.
+* When following the [standard install procedure](#msi), we recommend you enable **Instrument all**.
+* When you're done with the standard install instructions, you must explicitly enable the agent for that application (explained below). 
+* Because most non-IIS applications are not instrumented automatically, you'll likely need to [set up custom instrumentation](/docs/apm/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation) to get data reporting to New Relic. 
```

Comment:
maybe a link to the compatibility doc. it def is 'most'. some can be self-hosted outside of IIS.